src/pages/postList/composePost/__tests__/CreateGeneralPost.test.tsx

```diff
@@ -1,17 +1,47 @@
+/**
+ * @jest-environment jsdom
+ */
+
 import React from "react";
-import { shallow } from "enzyme";
+import { mount, shallow } from "enzyme";
```

Comment:
If mount is not used it would be ideal to remove it from imports to prevent linting warnings

Reply:
> If mount is not used it would be ideal to remove it from imports to prevent linting warnings

All unused mount methods have been removed